test/videojs.ima.test.js

```diff
@@ -58,3 +58,42 @@ test('ad plays on good ad tag', function() {
     start();
   }, 5000);
 });
+
+test('video continues after ad was skipped', function() {
+  var options = {
+    id: 'video',
+    adTagUrl: 'http://pubads.g.doubleclick.net/gampad/ads?sz=640x360&' +
+        'iu=/6062/iab_vast_samples/skippable&ciu_szs=300x250,728x90&impl=s&' +
+        'gdfp_req=1&env=vp&output=xml_vast2&unviewed_position_start=1&' +
+        'url=[referrer_url]&correlator=[timestamp]'
+  }
+
+  var readyForPrerollCallback = function() {
+    player.ima.addEventListener(google.ima.AdEvent.Type.SKIPPABLE_STATE_CHANGED, function() {
+      this.skip();
+    })
+    player.ima.start();
```

Comment:
start is normally bound as 'readyforpreroll'-listener when you don't pass in a callback to player.ima(). In our case we pass one and have to call start ourselves https://github.com/googleads/videojs-ima/blob/master/src/videojs.ima.js#L888-L891